lighthouse-core/gather/computed/critical-request-chains.js

```diff
@@ -38,6 +38,10 @@ class CriticalRequestChains extends ComputedArtifact {
     if (resourceTypeCategory === WebInspector.resourceTypes.XHR._category) {
       return false;
     }
+    // Images are also non-critical.
+    if (resourceTypeCategory === WebInspector.resourceTypes.Image._category) {
```

Comment:
I wonder if it would be cleaner to do an array of resourceTypes that are non-critical and check includes on it similar to request priority